tests/h/services/search_index/_queue_test.py

```diff
@@ -137,206 +136,192 @@ def add_where(self, queue):
 
 class TestSync:
     def test_it_does_nothing_if_the_queue_is_empty(self, batch_indexer, queue):
-        counts = queue.sync(LIMIT)
+        counts = queue.sync(1)
 
         assert counts == {}
         batch_indexer.index.assert_not_called()
 
     def test_it_ignores_jobs_that_arent_scheduled_yet(
-        self, add_all, batch_indexer, queue
+        self, batch_indexer, now, queue, SyncAnnotationJobFactory
     ):
-        add_all(schedule_in=ONE_WEEK_IN_SECONDS)
+        SyncAnnotationJobFactory(scheduled_at=now + datetime_.timedelta(hours=1))
 
-        counts = queue.sync(LIMIT)
+        counts = queue.sync(1)
 
         assert counts == {}
         batch_indexer.index.assert_not_called()
 
-    @pytest.mark.usefixtures("with_queued_annotations")
     def test_it_ignores_jobs_beyond_limit(
-        self, all_annotation_ids, batch_indexer, queue
+        self, batch_indexer, queue, SyncAnnotationJobFactory
     ):
-        queue.sync(LIMIT)
+        limit = 1
+        SyncAnnotationJobFactory.create_batch(size=limit + 1)
+
+        queue.sync(limit)
 
-        for annotation_id in all_annotation_ids[LIMIT:]:
-            assert annotation_id not in batch_indexer.index.call_args[0][0]
+        assert len(batch_indexer.index.call_args[0][0]) == limit
 
     def test_it_ignores_jobs_that_are_expired(
-        self, batch_indexer, db_session, factories, now, queue
+        self, batch_indexer, db_session, now, queue, SyncAnnotationJobFactory
     ):
-        annotation = factories.Annotation()
-        queue.add_by_id(annotation.id, "test_tag", schedule_in=MINUS_5_MIN_IN_SECS)
-        # Change the job's expires_at date so that it's expired.
-        job = db_session.query(Job).one()
-        job.expires_at = now - datetime_.timedelta(minutes=1)
+        job = SyncAnnotationJobFactory(expires_at=now - datetime_.timedelta(hours=1))
 
-        counts = queue.sync(LIMIT)
+        counts = queue.sync(1)
 
         assert counts == {}
         batch_indexer.index.assert_not_called()
         assert db_session.query(Job).all() == [job]
 
-    @pytest.mark.usefixtures("with_indexed_annotations")
     def test_if_the_job_has_force_True_it_indexes_the_annotation_and_deletes_the_job(
-        self, annotation_ids, add_all, batch_indexer, db_session, queue
+        self, batch_indexer, db_session, queue, SyncAnnotationJobFactory
     ):
-        add_all(force=True)
+        job = SyncAnnotationJobFactory(force=True)
 
-        counts = queue.sync(LIMIT)
+        counts = queue.sync(1)
 
         assert counts == {
-            Queue.Result.SYNCED_FORCED: LIMIT,
-            Queue.Result.COMPLETED_FORCED: LIMIT,
-            Queue.Result.SYNCED_TOTAL: LIMIT,
-            Queue.Result.COMPLETED_TOTAL: LIMIT,
+            Queue.Result.SYNCED_FORCED: 1,
+            Queue.Result.COMPLETED_FORCED: 1,
+            Queue.Result.SYNCED_TOTAL: 1,
+            Queue.Result.COMPLETED_TOTAL: 1,
```

Comment:
Again, the test added one job to the DB. It just needs to call `sync(1)`. There's no need to keep referring to `LIMIT` everywhere as if it might mean something important. It's just confusing.